game_data.py

```diff
@@ -1,12 +1,41 @@
-# structure defining the rooms in the game. Try adding more rooms to the game!
-rooms = {
-    "Tavern": {
-        "description": "You're in a cozy tavern warmed by an open fire.",
-        "exits": {"outside": "Outside"},
-    },
-    "Outside": {
-        "description": "You're standing outside a tavern. It's raining.",
-        "exits": {"inside": "Tavern"},
-    }
-}
+from lib.models.entity import Room
+from lib.models.enums import ExitType
 
+rooms = {}
```

Comment:
Did you consider moving to an eg `Rooms` class, to get this out of the global namespace?

Reply:
I will probably do that in the next iteration.